tests/Unit/Invoice/Domain/CallbackUrl/CallbackUrlTest.php

```diff
@@ -19,12 +19,17 @@ public function test_get_callback_url(): void
 
         $httpFacade = $this->createStub(HttpFacadeInterface::class);
 
-        $callbackUrl = new CallbackUrl($httpFacade, 'ln@address', 'https://domain/receiver');
+        $callbackUrl = new CallbackUrl(
+            $httpFacade,
+            SendableRange::withMinMax(1_000, 5_000),
+            'ln@address',
+            'https://domain/receiver',
+        );
 
-        self::assertSame([
+        self::assertEquals([
             'callback' => 'https://domain/receiver',
-            'maxSendable' => InvoiceGenerator::MAX_SENDABLE,
-            'minSendable' => InvoiceGenerator::MIN_SENDABLE,
+            'minSendable' => 1_000,
```

Comment:
Does it make sense to do some test where `min` is higher than `max` when the user defined these values in the config (or somewhere else)?

Reply:
Yeah, good point 👍🏼